### PR TITLE
add cube op for v0.6.2

### PR DESCRIPTION
**Pull request by bjjwwang**
# PR TYPE
new feature

# Describe
-  resume dist kv op
-  resume ctr with cube example
- resume cmake cube compile

### CODE REVIEW

**Comment by HexToString** [comment]
what a significant job you done

File: core/general-server/op/general_dist_kv_infer_op.cpp [hunk 1]
```diff
@@ -37,7 +37,147 @@ using baidu::paddle_serving::predictor::general_model::Request;
...
+               << ") can only have one predecessor op, but received "
+               << pre_node_names.size();
+    return -1;
+  }
+  const std::string pre_name = pre_node_names[0];
+
+  const GeneralBlob *input_blob = get_depend_argument<GeneralBlob>(pre_name);
+  if (!input_blob) {
+    LOG(ERROR) << "input_blob is nullptr,error";
+      return -1;
```
Comment (TeslaZhao) [inline core/general-server/op/general_dist_kv_infer_op.cpp]: 代码对齐

File: core/general-server/op/general_dist_kv_infer_op.cpp [hunk 2]
```diff
@@ -37,7 +37,147 @@ using baidu::paddle_serving::predictor::general_model::Request;
...
+  }
+  uint64_t log_id = input_blob->GetLogId();
+  VLOG(2) << "(logid=" << log_id << ") Get precedent op name: " << pre_name;
+
+  GeneralBlob *output_blob = mutable_data<GeneralBlob>();
+  if (!output_blob) {
+    LOG(ERROR) << "output_blob is nullptr,error";
+      return -1;
```
Comment (TeslaZhao) [inline core/general-server/op/general_dist_kv_infer_op.cpp:64]: 格式对齐

Comment (TeslaZhao) [reply core/general-server/op/general_dist_kv_infer_op.cpp:64]: LOG ERROR时打印出log_id

File: core/cube/cube-agent/src/CMakeLists.txt [hunk 3]
```diff
@@ -14,6 +14,7 @@
 
 set(SOURCE_FILE cube-agent.go)
 add_go_executable(cube-agent ${SOURCE_FILE})
+add_dependencies(cube-agent paddle_python)
```
Comment (TeslaZhao) [inline core/cube/cube-agent/src/CMakeLists.txt]: delete

File: core/cube/cube-transfer/src/CMakeLists.txt [hunk 4]
```diff
@@ -14,6 +14,7 @@
 
 set(SOURCE_FILE cube-transfer.go)
 add_go_executable(cube-transfer ${SOURCE_FILE})
-add_dependencies(cube-transfer docopt-go)
-add_dependencies(cube-transfer rfw)
-add_dependencies(cube-transfer logex)
+add_dependencies(cube-transfer paddle_python)
```
Comment (TeslaZhao) [inline core/cube/cube-transfer/src/CMakeLists.txt]: delete

File: core/general-server/op/general_dist_kv_infer_op.cpp [hunk 5]
```diff
@@ -37,7 +37,147 @@ using baidu::paddle_serving::predictor::general_model::Request;
 using baidu::paddle_serving::predictor::InferManager;
 using baidu::paddle_serving::predictor::PaddleGeneralModelConfig;
 
-int GeneralDistKVInferOp::inference() { return 0; }
+int GeneralDistKVInferOp::inference() {
```
Comment (TeslaZhao) [inline core/general-server/op/general_dist_kv_infer_op.cpp:42]: 增加一个注释说明

**Pull request merged**